currency.go

```diff
@@ -4,7 +4,7 @@ import (
 	"strings"
 )
 
-// Currency represents money currency information required for formatting.
+// Currency represents money currency information required for formatting
```

Comment:
See golang.org/doc/effective_go.html#commentary. Comments documenting declarations should be full sentences, even if that seems a little redundant. This approach makes them format well when extracted into godoc documentation. Comments should begin with the name of the thing being described and end in a period

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
yeah, not sure why I deleted them.. added them back